Tools/CMake/AMReXOptions.cmake

```diff
@@ -155,8 +155,14 @@ print_option(  AMReX_DPCPP_SPLIT_KERNEL )
 
 # --- HIP ----
 if (AMReX_HIP)
-   set(AMReX_AMD_ARCH "IGNORE" CACHE STRING
+   set(AMReX_AMD_ARCH_DEFAULT "IGNORE")
```

Comment:
Do we really need to require a user-input here?
@WeiqunZhang mentioned that without a flag, `hipcc` picks `803` these days? (Note that AMD build architectures are not forward compatible as of today.)

Reply:
I saw this if we don't specify amd gpu target
```2021-05-18T23:13:33.9776231Z /opt/rocm-4.1.1/hip/include/hip/hcc_detail/hip_runtime_api.h:55:41: note: expanded from macro 'DEPRECATED'
2021-05-18T23:13:33.9777132Z #define DEPRECATED(msg) __attribute__ ((deprecated(msg)))
2021-05-18T23:13:33.9777664Z                                         ^
2021-05-18T23:13:35.1172406Z 1 warning generated when compiling for gfx803.
```

Reply:
Since things are not forward compatible, we might continue to require a user-choice for the time being.